cartridge/stateboard-client.lua

```diff
@@ -253,12 +253,22 @@ local function longpoll(client, timeout)
     end
 end
 
+--- Check that stateboard is available.
+--
+-- @function check_quorum
+-- @treturn boolean true / false
+local function check_quorum(client)
```

Comment:
In case of stateboard there is just one instance of state provider. Thus, strictly speaking, there is no quorum but connectivity with this particular instance. 
`check_quorum` terminology originates from cases when there is etcd as a state provider and it can, in fact, consist of several nodes. `check_quorum` behaves in similar manner as a write operation and ensures that all parts of etcd cluster are in sync.
https://github.com/tarantool/cartridge/blob/d15d125e8b93254322344cca18b669d371ea9285/cartridge/etcd2-client.lua#L381

However, the similar names of the functions are made for the sake of similar API of different state providers.

Reply:
Thanks for explanation. Ok